### PR TITLE
update habitat logo files to remove diamond logo marks

### PR DESCRIPTION
**Pull request by apriofrost**
Signed-off-by: Liz Chen <apriofrost@gmail.com>
tenor-83068462

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by rsertelon** [comment]
Hey @apriofrost I'm wondering if https://github.com/habitat-sh/habitat/pull/4865/files?short_path=858b53e#diff-3c46486a6f898960c1691f058aab6a6b needs to be that big now, mayeb the size can be reduced?

**Comment by apriofrost** [comment]
Thanks for the feedback! @rsertelon you meant the one in `www/source/images/habitat-social.jpg`? I tried to keep the overall height and width similar to its existing one, in case something breaks. 
We should have another PR to clean up some unused files, and probably make the logo size consistent across (currently the sizes vary a lot). That should be included in the issue #4846

File: www/source/images/habitat-logo-birthday.svg [hunk 1]
```diff
@@ -1 +1,80 @@
...1.79Z"/><path class="cls-11" d="M183.65,6.08a11.78,11.78,0,0,1,22.1,5.71c0,16.51-22.1,28.52-22.1,28.52-.31-.21-18.08-12.15-18.08-28.52,0-6.17,3.87-11.23,8.81-11.74A11.78,11.78,0,0,1,183.65,6.08Z"/><path class="cls-1" d="M196,19.55l-3,3a.46.46,0,0,1-.79-.33V21.14a.46.46,0,0,0-.46-.46h-1.63a.46.46,0,0,0-.46.46v2.13a.46.46,0,0,0,.46.46h.55a.46.46,0,0,1,.33.79l-3,3a.46.46,0,0,1-.64,0l-3-3a.46.46,0,0,1,.33-.79h.64a.46.46,0,0,0,.46-.46v-2.1a.46.46,0,0,0-.46-.46h-3.57a.46.46,0,0,0-.46.46v2.1a.46.46,0,0,0,.46.46h.67a.46.46,0,0,1,.33.79l-3,3a.46.46,0,0,1-.64,0l-3-3a.46.46,0,0,1,.33-.79h.48a.46.46,0,0,0,.46-.46V21.14a.46.46,0,0,0-.46-.46h-1.61a.46.46,0,0,0-.46.46v1a.46.46,0,0,1-.79.33l-3-3a.46.46,0,0,1,0-.64l3-3a.46.46,0,0,1,.79.33v1a.46.46,0,0,0,.46.46h1.58a.46.46,0,0,0,.46-.46V15.16a.46.46,0,0,0-.46-.46h-.53a.46.46,0,0,1-.33-.79l3-3a.46.46,0,0,1,.64,0l3,3a.46.46,0,0,1-.33.79h-.67a.46.46,0,0,0-.46.46v2.08a.46.46,0,0,0,.46.46h3.57a.46.46,0,0,0,.46-.46V15.15a.46.46,0,0,0-.46-.46h-.64a.46.46,0,0,1-.33-.79l3-3a.46.46,0,0,1,.64,0l3,3a.46.46,0,0,1-.33.79h-.55a.46.46,0,0,0-.46.46v2.13a.46.46,0,0,0,.46.46h1.63a.46.46,0,0,0,.46-.46v-1a.46.46,0,0,1,.79-.33l3,3A.46.46,0,0,1,196,19.55Z"/></g></g></svg>
```
Comment (cnunciato) [inline www/source/images/habitat-logo-birthday.svg]: I'm not seeing where this logo is used (but it does look like `habitat-birthday-dark` is, for example here: http://localhost:4567/blog/2017/06/Happy-Birthday-Habitat).

File: www/source/images/habitat-mark.svg [hunk 2]
```diff
@@ -1,19 +1,12 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (cnunciato) [inline www/source/images/habitat-mark.svg]: Per our chat, let's just drop these and remove the references to them.

File: www/source/images/habitat-logo-birthday.svg (hunk 1 above)
Comment (apriofrost) [reply www/source/images/habitat-logo-birthday.svg]: Good catch!

File: www/source/images/habitat-mark.svg (hunk 2 above)
Comment (apriofrost) [reply www/source/images/habitat-mark.svg]: Will do!

File: www/source/images/solutions-logos/icon-habitat.svg [hunk 3]
```diff
@@ -1,6 +1,12 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="50" height="50" viewBox="0 0 50 50">
```
Comment (cnunciato) [inline www/source/images/solutions-logos/icon-habitat.svg]: Same here, we should just remove this, and its reference also (looks like there's only one, in the Node tutorial).

**Comment by apriofrost** [comment]
@ryankeairns FYI, I've removed the habitat logo mark reference in all the `/getting-started` pages.

File: components/common/src/command/package/install.rs [hunk 4]
```diff
@@ -816,8 +816,9 @@ impl<'a> InstallTask<'a> {
         // Canonicalize both paths to ensure that there aren't any symlinks when comparing them
         // later.
         let artifact_path = artifact_path.canonicalize()?;
-        let cache_path = self.cached_artifact_path(ident).canonicalize()?;
         fs::create_dir_all(self.artifact_cache_path)?;
+        let artifact_cache_path = self.artifact_cache_path.canonicalize()?;
+        let cache_path = artifact_cache_path.join(ident.archive_name());
```
Comment (baumanj) [inline components/common/src/command/package/install.rs]: I think you're going to need to rebase your PR so you don't accidentally revert https://github.com/habitat-sh/habitat/pull/4864

Let me know if you need any help with that

Comment (apriofrost) [reply components/common/src/command/package/install.rs]: Yeah. I have bad rebase on this and the other open PR. Chris is going to help me with it today. Thanks for pointing it out Jon!

**Pull request merged**